src/raster/r.forcircular/r.forcircular

```diff
@@ -0,0 +1,2039 @@
+        )
+
+    run_command("r.null", map="volume", null=0)
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="revenues = technical_surface*(volume*(perc_roun*"
+        + "pric_roun + perc_timb*pric_timb + perc_fire*pric_fire + "
+        + "perc_res*PCI*pric_bioe))",
+    )
+
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="fell_productHFtr1 = if(management ==1 && treatment"
+        + " ==1 && tree_diam <99999,(aerial_extraction*(42-(2.6*tree_diam))"
+        + "/(-20))*1.65*(1-(slope/100)), if(management ==1 && treatment ==1"
+        + " && tree_diam == 99999,(aerial_extraction*"
+        + "(42-(2.6*35))/(-20))*1.65*(1-(slope/100))))",
+    )
+
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="fell_productHFtr2 = if(management ==1 && treatment ==2"
+        + " && tree_diam <99999,(aerial_extraction*(42-(2.6*tree_diam))/"
+        + "(-20))*1.65*(1-(slope/100)), if(management ==1 && treatment ==2"
+        + " && tree_diam == 99999,(aerial_extraction*(42-(2.6*35))/"
+        + "(-20))*1.65*(1-(slope/100))))",
+    )
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="fell_proc_productC = if(management ==2 && soil_prod <99999"
+        + ",((0.3-(1.1*soil_prod))/(-4))*(1-(slope/100)), if(management ==2 &&"
+        + " soil_prod == 99999,((0.3-(1.1*3))/(-4))*(1-(slope/100))))",
+    )
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="proc_productHFtr1 = if(management == 1 && treatment == 1 "
+        + "&& tree_diam==99999, aerial_extraction*0.363*35^1.116, "
+        + "if(management == 1 && treatment == 1 && tree_diam<99999, "
+        + "aerial_extraction *0.363*tree_diam^1.116))",
+    )
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="fell_proc_productHFtr1 = if(management == 1 && treatment"
+        + " == 1 && tree_vol<9.999, HFground_extraction*60/(1.5*(2.71^(0.1480"
+        + "-0.3894*2+0.0002*(slope^2)-0.2674*2.5))+(1.0667+(0.3094/tree_vol)"
+        + "-0.1846*1)), if(management == 1 && treatment == 1 && "
+        + "tree_vol==9.999 , HFground_extraction*60/(1.5*(2.71^(0.1480-"
+        + "0.3894*2+0.0002*(slope^2)-0.2674*2.5))+(1.0667+(0.3094/0.7)"
+        + "-0.1846*1))))",
+    )
+    run_command(
+        "r.mapcalc",
+        overwrite=1,
+        expression="fell_proc_productHFtr2 = if(management@PERMANENT == 1 && "
+        + "treatment@PERMANENT == 2 && tree_vol<9.999, "
+        + "HFground_extraction@PERMANENT*60/(1.5*(2.71^(0.1480-"
+        + "0.3894*2+0.0002*(slope@PERMANENT^2)-0.2674*2.5))+(1.0667+"
+        + "(0.3094/tree_vol@PERMANENT)-0.1846*1))*0.8,if(management@PERMANENT"
+        + " == 1 && treatment@PERMANENT == 2 && "
+        + "tree_vol==9.999, HFground_extraction@PERMANENT*60/(1.5*"
+        + "(2.71^(0.1480-0.3894*2+0.0002*(slope@PERMANENT^2)-0.2674*2.5))"
```

Comment:
I think the `@PERMANENT` part of the map names should be removed

Reply:
Done